packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -306,6 +320,9 @@ class Scaffold extends StatefulWidget {
   /// Typically a [Drawer].
   final Widget drawer;
 
+  /// A bottom navigation bar to display at the bottom of the scaffold.
```

Comment:
This would be the place to explain how the bottom navigation bar is laid out vis the scaffold's bottom sheet and its snackbar.